package.json

```diff
@@ -79,7 +79,7 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-svelte": "^2.19.0",
     "eslint-plugin-svelte3": "^3.4.1",
-    "flexsearch": "^0.7.31",
+    "flexsearch": "0.7.31",
```

Comment:
Thanks @fredhohman!! 🙏 I just noticed that we should probably also freeze `@types/flexsearch`.